src/d3a/d3a_core/simulation.py

```diff
@@ -339,6 +339,9 @@ def _execute_simulation(self, slot_resume, tick_resume, console=None):
             if self.simulation_config.external_connection_enabled:
                 external_global_statistics.update(market_cycle=True)
                 self.area.publish_market_cycle_to_external_clients()
+                if is_custom_matching_enabled():
+                    self.simulation_config.bid_offer_matcher.\
+                        match_algorithm.publish_market_cycle_myco()
```

Comment:
Nit: to follow PEP-8 and avoid using backslashes you could use parentheses:
```suggestion
                if is_custom_matching_enabled():
                    (self.simulation_config.bid_offer_matcher
                        .match_algorithm.publish_market_cycle_myco())
```